src/zed_rope.ml

```diff
@@ -722,9 +723,16 @@ module Zip_raw = struct
     zip : rope_zipper;
   }
 
+  (* let rec pp_rope t=
+    match t with
+    | Leaf (s,_)-> s
+    | Node (_,_,l,_,r)-> Zed_string.append (pp_rope l) (pp_rope r) *)
+
   let rec make_f_rec ofs rope pos rest_b rest_f =
+    (* Format.printf "ROPE == %s\n" (Zed_string.to_utf8 (pp_rope rope)); *)
```

Comment:
Same as above: you can remove the debugging code.

Reply:
Ack!